From 9a06f45ec71116d76ee4b268ebe1b33d45b06fc0 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 16 Apr 2016 22:10:43 -0400
Subject: [PATCH mtd-utils] include sys/sysmacros.h for major/minor/makedev

These functions have always been defined in sys/sysmacros.h under
Linux C libraries.  For some, including sys/types.h implicitly
includes that as well, but glibc wants to deprecate that, and some
others already have.  Include the header explicitly for the funcs.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 include/common.h | 1 +
 1 file changed, 1 insertion(+)

--- a/include/common.h
+++ b/include/common.h
@@ -27,6 +27,7 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <inttypes.h>
+#include <sys/sysmacros.h>
 #include "version.h"
 
 #ifndef PROGRAM_NAME
